src/Utils/calculateScore.tsx

```diff
@@ -0,0 +1,14 @@
+import { Option, Question } from "../Data/quiz.types";
```

Comment:
The files which do not contain jsx should better be named as .ts or .js only so that it's clear what the file contains.